pkg/cedar/core/threat.go

```diff
@@ -64,7 +65,7 @@ func (c *Client) GetThreat(ctx context.Context, cedarSystemID string) ([]*models
 		retVal = append(retVal, &models.CedarThreat{
 			AlternativeID:     zero.StringFrom(threat.AlternativeID),
 			ControlFamily:     zero.StringFrom(threat.ControlFamily),
-			DaysOpen:          zero.IntFrom(int64(threat.DaysOpen)),
+			DaysOpen:          null.IntFrom(int64(threat.DaysOpen)),
```

Comment:
Two questions based on the [generated model|https://github.com/CMSgov/easi-app/blob/5c332134e1b283caaebb1ef73214a68e3cba7fb3/pkg/cedar/core/gen/models/threat.go#L27]: 

1. Is there any benefit here of using `null` to convert the value when the underlying client is converting to `0` if the value is missing based on `DaysOpen` being an `int32`?
2. Should `int64` be `int32` here since that's what the code gen uses?

Reply:
Yeah, I think it'd be nice to take a pass at the overall types we use on this (and maybe our Codegen tooling too...)

For now, I think it should be safe, and passing a int32 -> int64 should be safe (just the other way around that I'd worry!)